crc32.c

```
@@ -10,35 +10,61 @@
 #include "zbuild.h"
 #include "functable.h"
 #include "crc32_braid_tbl.h"
+#include "crc32_p.h"
 
 /* ========================================================================= */
 
-const uint32_t * Z_EXPORT PREFIX(get_crc_table)(void) {
+Z_EXPORT const uint32_t * PREFIX(get_crc_table)(void) {
```

Cppcheck (2.20.0)

[style] The function 'zng_get_crc_table' is never used.

(unusedFunction)